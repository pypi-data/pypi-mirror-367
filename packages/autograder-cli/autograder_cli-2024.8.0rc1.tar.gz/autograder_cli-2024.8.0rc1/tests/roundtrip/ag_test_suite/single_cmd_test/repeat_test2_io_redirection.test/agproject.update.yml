project:
  name: Test Project c8115a2e466348838a2cd22e805edb61
  timezone: America/Chicago
  course:
    name: Test Course
    semester: Summer
    year: 2014
  settings:
  test_suites:
  - name: Suite
    test_cases:
    - name: Test $test_name
      cmd: echo $test_arg wee
      return_code:
        expected: zero
      input:
        source: instructor_file
        instructor_file: $input
      stdout:
        compare_with: instructor_file
        instructor_file: $correct_stdout
      stderr:
        compare_with: instructor_file
        instructor_file: $correct_stderr
      repeat:
        - $test_name: Spam
          $test_arg: test spam
          $input: input2.txt
          $correct_stdout: output2.txt
          $correct_stderr: stderr2.txt
        - $test_name: Egg
          $test_arg: test eggg
          $input: input3.txt
          $correct_stdout: output3.txt
          $correct_stderr: stderr3.txt
        - $test_name: Sausage
          $test_arg: test suasage
          $input: input1.txt
          $correct_stdout: output1.txt
          $correct_stderr: stderr1.txt
