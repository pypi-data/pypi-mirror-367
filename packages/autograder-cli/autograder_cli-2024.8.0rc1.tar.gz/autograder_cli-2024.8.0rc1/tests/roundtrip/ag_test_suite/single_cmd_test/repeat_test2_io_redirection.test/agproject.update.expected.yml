project:
  name: Test Project c8115a2e466348838a2cd22e805edb61
  timezone: America/Chicago
  course:
    name: Test Course
    semester: Summer
    year: 2014
  settings: {}
  instructor_files:
  - local_path: input1.txt
  - local_path: input2.txt
  - local_path: input3.txt
  - local_path: output1.txt
  - local_path: output2.txt
  - local_path: output3.txt
  - local_path: stderr1.txt
  - local_path: stderr2.txt
  - local_path: stderr3.txt
  test_suites:
  - name: Suite
    test_cases:
    - name: Test Spam
      cmd: echo test spam wee
      input:
        source: instructor_file
        instructor_file: input2.txt
      return_code:
        expected: zero
      stdout:
        compare_with: instructor_file
        instructor_file: output2.txt
      stderr:
        compare_with: instructor_file
        instructor_file: stderr2.txt
    - name: Test Egg
      cmd: echo test eggg wee
      input:
        source: instructor_file
        instructor_file: input3.txt
      return_code:
        expected: zero
      stdout:
        compare_with: instructor_file
        instructor_file: output3.txt
      stderr:
        compare_with: instructor_file
        instructor_file: stderr3.txt
    - name: Test Sausage
      cmd: echo test suasage wee
      input:
        source: instructor_file
        instructor_file: input1.txt
      return_code:
        expected: zero
      stdout:
        compare_with: instructor_file
        instructor_file: output1.txt
      stderr:
        compare_with: instructor_file
        instructor_file: stderr1.txt
